conduit-extra/Data/Conduit/Binary.hs

```diff
@@ -181,7 +182,20 @@ sourceHandleRange :: MonadIO m
                   -> Maybe Integer -- ^ Offset
                   -> Maybe Integer -- ^ Maximum count
                   -> Producer m S.ByteString
-sourceHandleRange handle offset count = do
+sourceHandleRange handle offset count =
+  sourceHandleRangeWithBuffer handle offset count 4096
```

Comment:
Ouch, having the buffer size so low in the first place was a big mistake. I'll fix that also.